omnibus/config/software/datadog-iot-agent.rb

```diff
@@ -55,33 +55,27 @@
     copy 'bin/agent', "#{install_dir}/bin/"
 
     # Upstart
-    if debian_target?
-      erb source: "upstart_debian.conf.erb",
-          dest: "/etc/init/datadog-agent.conf",
-          mode: 0644,
-          vars: { install_dir: install_dir, etc_dir: etc_dir }
-    elsif redhat_target? || suse_target?
-      # Ship a different upstart job definition on RHEL to accommodate the old
-      # version of upstart (0.6.5) that RHEL 6 provides.
-      erb source: "upstart_redhat.conf.erb",
-          dest: "/etc/init/datadog-agent.conf",
-          mode: 0644,
-          vars: { install_dir: install_dir, etc_dir: etc_dir }
-    end
+    erb source: "upstart_debian.conf.erb",
+        dest: "/etc/init/datadog-agent.conf",
+        mode: 0644,
+        vars: { install_dir: install_dir, etc_dir: etc_dir }
+    # Ship a different upstart job definition on RHEL to accommodate the old
+    # version of upstart (0.6.5) that RHEL 6 provides.
+    erb source: "upstart_redhat.conf.erb",
+        dest: "/etc/init/datadog-agent.conf",
+        mode: 0644,
+        vars: { install_dir: install_dir, etc_dir: etc_dir }
```

Comment:
Doesn't that mean the redhat init script is always going to be used, as it will overwrite the debian one? I think this should follow the Agent example, and be placed in a separate `init-scripts-iot-agent` definition run at packaging time?

Reply:
indeed, nice catch!